bonsai/BudgetTransactions/BudgetTransactions.swift

```diff
@@ -0,0 +1,127 @@
+//
+//  BudgetTransactions.swift
+//  bonsai
+//
+//  Created by hoang on 08.12.2021.
+//
+
+import SwiftUI
+
+struct VisualEffectView: UIViewRepresentable {
+    var effect: UIVisualEffect?
+    func makeUIView(context: UIViewRepresentableContext<Self>) -> UIVisualEffectView { UIVisualEffectView() }
+    func updateUIView(_ uiView: UIVisualEffectView, context: UIViewRepresentableContext<Self>) { uiView.effect = effect }
+}
+
+struct TransactionData: Identifiable {
+    let id = UUID()
+    let dateTitle: String
+    let items: [TransactionItem]
+}
+
+struct TransactionItem: Identifiable, Hashable {
+    let id = UUID()
+    let image: UIImage
+    let name: String
+    let value: Double
+}
+
+struct RoundedCorner: Shape {
+
+    var radius: CGFloat = .infinity
+    var corners: UIRectCorner = .allCorners
+
+    func path(in rect: CGRect) -> Path {
+        let path = UIBezierPath(roundedRect: rect, byRoundingCorners: corners, cornerRadii: CGSize(width: radius, height: radius))
+        return Path(path.cgPath)
+    }
+}
+
+extension View {
+    func cornerRadius(_ radius: CGFloat, corners: UIRectCorner) -> some View {
+        clipShape(RoundedCorner(radius: radius, corners: corners))
+    }
+}
+
+
+struct BudgetTransactions: View {
+    
+    let transactions = [
+        TransactionData(dateTitle: "Today", items: [
+            TransactionItem(image: BonsaiImage.amount,
+                            name: "Okko", value: 25.0),
+            TransactionItem(image: BonsaiImage.tag,
+                            name: "MacDonalz", value: 55.6)
+        ]),
+        TransactionData(dateTitle: "25.12.2021", items: [
+            TransactionItem(image: BonsaiImage.amount,
+                            name: "KEK", value: 25.0),
+            TransactionItem(image: BonsaiImage.tag,
+                            name: "LOL", value: 55.6)
+        ]),
+        TransactionData(dateTitle: "24.12.2021", items: [
+            TransactionItem(image: BonsaiImage.amount,
+                            name: "AZAZA", value: 25.0),
+            TransactionItem(image: BonsaiImage.tag,
+                            name: "KANAVA", value: 55.6),
+        ])
+    ]
+    
+    var body: some View {
+        ZStack {
+            VisualEffectView(
+                effect: UIBlurEffect(style: .dark))
+                .edgesIgnoringSafeArea(.all)
+            
+            VStack(alignment: .leading) {
+                HStack {
+                    Spacer()
+                    RoundedCorner()
+                        .foregroundColor(BonsaiColor.disabled)
+                        .frame(width: 60, height: 5, alignment: .center)
+                        .cornerRadius(25, corners: .allCorners)
+                    Spacer()
+                }
+                
+                Text("Transactions")
+                    .font(BonsaiFont.title_headline_17)
```

Comment:
not sure if we should use Text or NavigationView with title here

Reply:
Simply text i think should be enough